src/lib/components/player/player-row.svelte

```diff
@@ -24,6 +26,13 @@
    <div class="rank" width="5px">
       <span class="rank">#{player.rank.toLocaleString('en-US')}</span>
    </div>
+   {#if countryFiltered}
```

Comment:
I don't think country rank should be in it's own element, doing so causes unwanted padding to the name which should always be quite close to the left, it should be appended to the rank element if applicable in the following format `#CountryRank (#GlobalRank)` not what it currently is `#GlobalRank (#CountryRank)`

Reply:
This is the only thing that can be done with also having the columns lined up, what can be done is changing the px count for the first column to set a smaller or larger space before the ellipsis start appearing. But 100px as has to be this long before it starts appearing 
![image](https://user-images.githubusercontent.com/4028289/143013520-84b36e7d-e426-450a-b329-248ab791d6fd.png)